project/MimaBuild.scala

```diff
@@ -88,7 +88,7 @@ object MimaBuild {
 
   def mimaSettings(sparkHome: File, projectRef: ProjectRef) = {
     val organization = "org.apache.spark"
-    val previousSparkVersion = "2.2.0"
+    val previousSparkVersion = "2.4.0"
```

Comment:
According to https://issues.apache.org/jira/browse/SPARK-23070, `2.3.0` seems to be the correct version in this PR.

cc @gatorsmile .

Reply:
up to my understanding, we should have changed it to 2.3.0 when 2.3.0 was released.

Maybe we should send a another PR to branch-2.4 and change it to 2.3.0

Reply:
I believe @cloud-fan is right here. If master is 3.0, the previous version is 2.4.0 in master. Analogously for branch 2.4. Yes sounds like this should be a step in the release process somewhere.

Reply:
Thanks, @cloud-fan and @srowen . +1 for another PR for `branch-2.4`.